simulator/simulated_robot.md

```diff
@@ -26,9 +26,10 @@ The main differences are:
 
 ## Motors
 
-Your robot has one motor board attached, the left wheel is connected to the first port, and the right wheel to the second.
+Your robot has one motor board attached, the motor on the left wheel is connected to the 'Motor 0' port, and the right wheel to 'Motor 1'. These can be referenced with `robot.motor_board.motors[0]`, and 
+`robot.motor_board.motors[1]`, respectively. See [the motor board programming docs]({{ site.baseurl }}/programming/motors) for how to control these.
```

Comment:
we seem to (mostly?) use double quotes for quoting, some whitespace nits:
```suggestion
Your robot has one motor board attached, the motor on the left wheel is connected to the "Motor 0" port, and the right wheel to "Motor 1". These can be referenced with `robot.motor_board.motors[0]`, and `robot.motor_board.motors[1]`, respectively.
See [the motor board programming docs]({{ site.baseurl }}/programming/motors) for how to control these.
```

Reply:
https://github.com/srobo/docs/pull/557/commits/fd1d5a325f0a25f82ed502bf1010dc0172b01b9e